sellercenter-product-port/src/main/java/com/gfg/sellercenter/product/port/service/ProductReaderService.java

```diff
@@ -29,7 +30,8 @@
     @NotNull
     private final JsonHttpReader jsonReader;
 
-    private static final String API_PATH = "/api/products/v1";
+    private static final String API_PATH_PRODUCTS_BY_IDS = "/api/products/v1";
+    private static final String API_PATH_SEARCH_PRODUCTS_OF_SELLER = "/api/sellers/v1/";
```

Comment:
do you mind to have similar URLs, i mean `../api/products/v1` also for `/api/sellers/v1` without slash in the end

Reply:
Ehm, it is possible, but schema of URLs is different. In first case it is the whole url (`/api/products/v1`). In other `/api/sellers/v1/` is first part of bigger thing `/api/sellers/v1/{sellerId}/products`. But no problem, I will move slash to other part of it

Reply:
Slash was from value of constant